apps/sparo-lib/src/cli/commands/pull.ts

```diff
@@ -0,0 +1,95 @@
+import { inject } from 'inversify';
+import { Command } from '../../decorator';
+import { GitService } from '../../services/GitService';
+import { SparoProfileService } from '../../services/SparoProfileService';
+
+import type { Argv, ArgumentsCamelCase } from 'yargs';
+import type { ICommand } from './base';
+import type { TerminalService } from '../../services/TerminalService';
+
+export interface IPullCommandOptions {
+  branch?: string;
+  remote?: string;
+  profile?: string[];
+  addProfile?: string[];
+}
+
+@Command()
+export class PullCommand implements ICommand<IPullCommandOptions> {
+  public cmd: string = 'pull [remote] [branch]';
+  public description: string = 'pull changes from remote branch to local';
```

Comment:
1. The first character usually needs to be capitalized
2. I run `git pull --help` to get the idea about the description. Do you think we can set description to `Incorporates changes from a remote repository into the current branch.` here?

```
GIT-PULL(1)                                                               Git Manual                                                              GIT-PULL(1)

NAME
       git-pull - Fetch from and integrate with another repository or a local branch

SYNOPSIS
       git pull [<options>] [<repository> [<refspec>...]]


DESCRIPTION
       Incorporates changes from a remote repository into the current branch. If the current branch is behind the remote, then by default it will
       fast-forward the current branch to match the remote. If the current branch and the remote have diverged, the user needs to specify how to reconcile
       the divergent branches with --rebase or --no-rebase (or the corresponding configuration option in pull.rebase).

       More precisely, git pull runs git fetch with the given parameters and then depending on configuration options or command line flags, will call either
       git rebase or git merge to reconcile diverging branches.

       <repository> should be the name of a remote repository as passed to git-fetch(1). <refspec> can name an arbitrary remote ref (for example, the name of
       a tag) or even a collection of refs with corresponding remote-tracking branches (e.g., refs/heads/*:refs/remotes/origin/*), but usually it is the name
       of a branch in the remote repository.

       Default values for <repository> and <branch> are read from the "remote" and "merge" configuration for the current branch as set by git-branch(1)
       --track.

       Assume the following history exists and the current branch is "master":

                     A---B---C master on origin
                    /
               D---E---F---G master
                   ^
                   origin/master in your repository


       Then "git pull" will fetch and replay the changes from the remote master branch since it diverged from the local master (i.e., E) until its current
       commit (C) on top of master and record the result in a new commit along with the names of the two parent commits and a log message from the user
       describing the changes.

                     A---B---C origin/master
                    /         \
               D---E---F---G---H master
```

Reply:
+1 Using correct grammar and business writing really does have a significant effect on users' confidence in adopting a tool.  It is less important in source code and log files, but it is very important in the UX (TUI console output without `--debug`), official docs, and any public APIs.

Reply:
Of course yes.

Reply:
> 1. The first character usually needs to be capitalized
> 2. I run `git pull --help` to get the idea about the description. Do you think we can set description to `Incorporates changes from a remote repository into the current branch.` here?
> 
> ```
> GIT-PULL(1)                                                               Git Manual                                                              GIT-PULL(1)
> 
> NAME
>        git-pull - Fetch from and integrate with another repository or a local branch
> 
> SYNOPSIS
>        git pull [<options>] [<repository> [<refspec>...]]
> 
> 
> DESCRIPTION
>        Incorporates changes from a remote repository into the current branch. If the current branch is behind the remote, then by default it will
>        fast-forward the current branch to match the remote. If the current branch and the remote have diverged, the user needs to specify how to reconcile
>        the divergent branches with --rebase or --no-rebase (or the corresponding configuration option in pull.rebase).
> 
>        More precisely, git pull runs git fetch with the given parameters and then depending on configuration options or command line flags, will call either
>        git rebase or git merge to reconcile diverging branches.
> 
>        <repository> should be the name of a remote repository as passed to git-fetch(1). <refspec> can name an arbitrary remote ref (for example, the name of
>        a tag) or even a collection of refs with corresponding remote-tracking branches (e.g., refs/heads/*:refs/remotes/origin/*), but usually it is the name
>        of a branch in the remote repository.
> 
>        Default values for <repository> and <branch> are read from the "remote" and "merge" configuration for the current branch as set by git-branch(1)
>        --track.
> 
>        Assume the following history exists and the current branch is "master":
> 
>                      A---B---C master on origin
>                     /
>                D---E---F---G master
>                    ^
>                    origin/master in your repository
> 
> 
>        Then "git pull" will fetch and replay the changes from the remote master branch since it diverged from the local master (i.e., E) until its current
>        commit (C) on top of master and record the result in a new commit along with the names of the two parent commits and a log message from the user
>        describing the changes.
> 
>                      A---B---C origin/master
>                     /         \
>                D---E---F---G---H master
> ```

Of course yes, let me change it.